docs/docker.rst

```diff
@@ -13,8 +13,8 @@ Images
 
 `Rasa NLU docker images <https://hub.docker.com/r/rasa/rasa_nlu/tags/>`_ are provided for different backends:
 
-- ``spacy``: If you use the :ref:`section_spacy_pipeline` pipeline
-- ``tensorflow``: If you use the :ref:`section_tensorflow_embedding_pipeline`
+- ``spacy``: If you use the :ref:`section_pretrained_embeddings_spacy_pipeline` pipeline
```

Comment:
this is then `spacy_en` and `spacy_de` right?

Reply:
To be consistent, we should rename these to be consistent with the pipeline names, so would be `pretrained_embeddings_spacy_{en, de}`, `supervised_embeddings`, etc.
However these are quite long for tag names, so we could stick with short names like `spacy_en`, `tensorflow`, etc